guides/rest_doc.md

```diff
@@ -741,6 +745,150 @@ kintoneUtility.rest.getFormLayout({
 });
 ```
 
+## <a name="postDeploy"> kintoneUtility.rest.postDeploy(param)
```

Comment:
I think the function name should be more clear
+ postDeploy => postDeployAppSettings